### ISSUE TITLE
JSONP Support

### ISSUE THREAD

**Issue by an0o0nym** [sdwolfe32/trumail#21]
Is there any chance for implementation of jsonp support?

**Comment by sdwolfe32** [comment]
Yeah sure! I recently switched to a self-written http framework but recently found labstacks echo which I think I'll use for this project instead. I'll be sure to add jsonp as an option!
